Cabal/Distribution/Simple/GHC/IPIConvert.hs

```diff
@@ -9,31 +9,31 @@
 --
 -- Helper functions for 'Distribution.Simple.GHC.IPI642'.
 module Distribution.Simple.GHC.IPIConvert (
-    PackageIdentifier, convertPackageId,
+    MungedId, convertMungedId,
     License, convertLicense,
     convertModuleName
   ) where
 
 import Prelude ()
 import Distribution.Compat.Prelude
 
-import qualified Distribution.Types.PackageId as Current
-import qualified Distribution.Types.PackageName as Current
+import qualified Distribution.Types.MungedId as Current
+import qualified Distribution.Types.MungedName as Current
 import qualified Distribution.License as Current
 
 import Distribution.Version
 import Distribution.ModuleName
 import Distribution.Text
 
-data PackageIdentifier = PackageIdentifier {
-    pkgName    :: String,
-    pkgVersion :: Version
+data MungedId = MungedId {
```

Comment:
Can we change the type name as long as the constructor stays the same? I'm trying that.